src/main/scala/uk/gov/hmrc/govukfrontend/views/Implicits.scala

```diff
@@ -54,16 +54,33 @@ trait Implicits {
 
     def asErrorLinks: Seq[ErrorLink] =
       formErrors.map { error =>
-        ErrorLink(href = Some(s"#${error.key}"), content = Text(messages(error.message, error.args: _*)))
+        ErrorLink(href = Some(s"#${error.key}"), content = Text(errorMessage(error)))
       }
 
+    def asErrorLinks(isContentHtml: Boolean): Seq[ErrorLink] =
```

Comment:
There is duplication in the code that could be removed with something like (untested):

```scala
    private def asErrorLinks(contentConstructor: String => Content): Seq[ErrorLink] =
      errors.map { error =>
        ErrorLink(href = Some(error.key), content = contentConstructor(messages(error.message, error.args: _*)))
      }

    def asHtmlErrorLinks: Seq[ErrorLink] =
      asErrorLinks(HtmlContent.apply)

    def asTextErrorLinks: Seq[ErrorLink] =
      asErrorLinks(Text.apply)
```